File: src/my_project_typescript/easy_problems_typescript/from51To100/designHashmap.ts
class MyHashMap {
    hashMap: Array<any> = []
    constructor() {

    }

    put(key: number, value: number): void {
        let flag: boolean = false 
        for (let i = 0; i < this.hashMap.length; i++){
            if (this.hashMap[i][0] == key){
                this.hashMap[i][1] = value 
                flag = true 
            }
        }
        if (flag == false){
            this.hashMap.push([key, value])

    }
}

    get(key: number): number {
        for (let i = 0; i < this.hashMap.length; i++){
            if (this.hashMap[i][0] == key){
                return this.hashMap[i][1]
            }
        }
        
        return -1
    }

    remove(key: number): void {
        for (let i = 0; i < this.hashMap.length; i++){
            if (this.hashMap[i][0] == key){
                this.hashMap = this.hashMap.slice(0,i).concat(this.hashMap.slice(i+1))
                break
            }

        }

    }
}

File: src/my_project/easy_problems/from151To200/design_hashmap.py
from typing import List 

class MyHashMap(object):

    def __init__(self):
        self.hashMap = []
        

    def put(self, key, value):
        flag = False
        for i in range(len(self.hashMap)):
            if self.hashMap[i][0] == key:
                self.hashMap[i][1] = value
                flag = True
        if flag == False: 
            self.hashMap.append([key,value])
        return None 
        """
        :type key: int
        :type value: int
        :rtype: None
        """
        

    def get(self, key):
        for i in range(len(self.hashMap)):
            if self.hashMap[i][0] == key:
                return self.hashMap[i][1]
        else:
            return -1
        """
        :type key: int
        :rtype: int
        """
        

    def remove(self, key):
        for i in range(len(self.hashMap)):
            if self.hashMap[i][0] == key:
                self.hashMap = self.hashMap[0:i] + self.hashMap[i+1:]
                break
        return None 
        """
        :type key: int
        :rtype: None
        """
